main/glossary/README.md

```diff
@@ -109,27 +107,26 @@ The installation operation returns
 an `installation`, which is an object with one method; `getBundle()`. You can access an installed contract's source
 code via `const { source } = await E(installation).getBundle();`.
```

Comment:
This probably needs to change to `const { endoZipBase64 } = await E(installation).getBundle(); console.log(endoZipBase64)` and down below `echo "$endoZipBase64" | base64 -d > bundle.zip` for clarity about how this transits from JS to shell.